coderd/httpmw/logger.go

```diff
@@ -26,19 +26,28 @@ func Logger(log slog.Logger) func(next http.Handler) http.Handler {
 				slog.F("path", r.URL.Path),
 				slog.F("proto", r.Proto),
 				slog.F("remote_addr", r.RemoteAddr),
+				// Include the start timestamp in the log so that we have the
+				// source of truth. There is at least a theoretical chance that
+				// there can be a delay between `next.ServeHTTP` ending and us
+				// actually logging the request. This can also be useful when
+				// filtering logs that started at a certain time (compared to
+				// trying to compute the value).
+				slog.F("start_timestamp", start),
```

Comment:
nit: `_timestamp` indicates a UNIX timestamp in milliseconds to me. If we're just logging a string, just name it `start`.